CenterlineMetrics/CenterlineMetrics.py

```diff
@@ -127,7 +138,51 @@ def onRadioRAS(self):
     
   def onDistinctCoordinatesCheckBox(self):
     self.logic.coordinateSystemColumnSingle = not self.ui.distinctColumnsCheckBox.checked
-
+  
+  def onJumpCentredInSliceNodeCheckBox(self):
+    self.logic.jumpCentredInSliceNode = self.ui.jumpCentredInSliceNodeCheckBox.checked
+    
+  def moveSliceView(self, value):
+    self.logic.moveSliceView(self.ui.sliceViewSelector.currentNode(), value)
+    self.updateUIWithMetrics(value)
+  
+  def updateUIWithMetrics(self, value):
+    tableNode = self.logic.outputTableNode
+    if tableNode is None:
+        return
+    coordinateArray = self.logic.getRASCoordinatesAtIndex(value)
+    distance = str(round(tableNode.GetTable().GetValue(int(value), 0).ToDouble(), 2)) + " mm"
```

Comment:
It would be better not to hardcode the unit, but retrieve the formatted length string from the unit node:

```python
>>> selectionNode = slicer.mrmlScene.GetNodeByID("vtkMRMLSelectionNodeSingleton")
>>> unitNode = slicer.mrmlScene.GetNodeByID(selectionNode.GetUnitNodeID("length"))
>>> unitNode.GetDisplayStringFromValue(34.1)
' 34.1000 mm'
```

Reply:
I could not get to a similar trick to format orientation in degrees. Is there a key to get an orientation unit node ?

Reply:
Since angle is always between 0-360 and unit is always degree, there is no unit node for angle by default.